irohad/torii/impl/command_service.cpp

```diff
@@ -270,7 +264,8 @@ namespace torii {
                   .build()
                   .getTransport();
             }())));
-    return responses_
+    return status_bus_
+        ->statuses()
```

Comment:
Is this code is well-formed with clang-format?

Reply:
yes